VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -8,6 +8,9 @@ namespace VulnerableApplication.Controllers
 {
     public class HomeController : Controller
     {
+        public static string SonarPassword = "foo";
+        public static string GitHubPassword = "bar";
```

Comment:
[CODE_SMELL] Change the visibility of 'SonarPassword' or make it 'const' or 'readonly'.